solutions/kueue-admission-check/setup-env.sh

```
@@ -30,19 +30,24 @@
   kind create cluster --name "$cluster" --image kindest/node:v1.29.0
 done
 
-echo "Initialize the ocm hub cluster with ClusterProfile enabled"
-clusteradm init --feature-gates="ManifestWorkReplicaSet=true,ManagedClusterAutoApproval=true,ClusterProfile=true" --bundle-version="v0.15.0" --wait --context ${hubctx}
+echo "Initialize the ocm hub cluster"
+clusteradm init --feature-gates="ManifestWorkReplicaSet=true,ManagedClusterAutoApproval=true" --bundle-version="v1.0.0" --wait --context ${hubctx}
 joincmd=$(clusteradm get token --context ${hubctx} | grep clusteradm)
 
 echo "Join clusters to hub"
+$(echo ${joincmd} --force-internal-endpoint-lookup --wait --context ${hubctx} | sed "s/<cluster_name>/$hub/g")
```

Shellcheck (0.10.0)

[warning] Remove surrounding $() to avoid executing output (or use eval if intentional).

(SC2091)